src/stories/definitionListData.ts

```diff
@@ -0,0 +1,8 @@
+const definitionListData: Record<string, React.ReactNode> | undefined = {
```

Comment:
Think the type is infered here. So could be: `const definitionListData = {`. Or `const definitionListData: Record<string, React.ReactNode> = {`

Reply:
True, removed it